include/engine/api/route_api.hpp

```diff
@@ -228,42 +228,50 @@ class RouteAPI : public BaseAPI
 
         std::vector<util::json::Object> annotations;
 
-        if (parameters.annotations_type != RouteParameters::AnnotationsType::None)
+        if (parameters.annotations_type != RouteParameters::AnnotationsType::None ||
+            parameters.annotations == true)
```

Comment:
Ah I get your point. Like, it should be good for how we use it now but better safe than sorry?